src/components/TransactionDetails/transactionTransformer.ts

```
@@ -318,7 +318,13 @@ export function mapTransactionDataForDrawer(entry: HistoryEntry): MappedTransact
         case EHistoryEntryType.DEPOSIT:
             direction = 'add'
             transactionCardType = 'add'
-            nameForDetails = entry.senderAccount?.identifier || 'Deposit Source'
+            // check if this is a test transaction (0 amount deposit during account setup), ideally this should be handled in the backend, but for now we'll handle it here cuz its a quick fix, and in promisland of post devconnect this should be handled in the backend.
+            const isTestTransaction = String(entry.amount) === '0' || entry.extraData?.usdAmount === '0'
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)